From 3d011d355b12998c42348622c88d104d1d14fa66 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Fri, 20 Oct 2017 23:57:21 +0800
Subject: [PATCH 2673/5242] MLK-16676-5 ARM64: dts: freescale: imx8qxp: add
 debug_console property

commit  bc45c851b57972a569d16dcb080c68a11c768c1d from
https://source.codeaurora.org/external/imx/linux-imx.git

Pass debug_console port info for power domain awareness.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    4 ++++
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |    4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 8ed3e29..14d6faf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -558,6 +558,10 @@
         };
 };
 
+&pd_dma_lpuart0 {
+	debug_console;
+};
+
 &lpuart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lpuart0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 1b8edf2..3d342c3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -434,6 +434,10 @@
 	};
 };
 
+&pd_dma_lpuart0 {
+	debug_console;
+};
+
 &lpuart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lpuart0>;
-- 
1.7.9.5

